Tests/Rules/PSCredentialType.tests.ps1

```diff
@@ -1,5 +1,5 @@
 ﻿Import-Module PSScriptAnalyzer 
-$violationMessage = "The Credential parameter in 'Credential' must be of the type PSCredential with CredentialAttribute where PSCredential comes before CredentialAttribute."
+$violationMessage = "The Credential parameter in 'Credential' must be of type PSCredential and must have a CredentialAttribute attribute such that PSCredential is placed before CredentialAttribute. This is not applicable to PowerShell version 5.0 or above."
```

Comment:
The Credential parameter in 'Credential' must be of type PSCredential. In PowerShell 4.0 and earlier, the Credential parameter must also have a Credential transformation attribute defined after the PSCredential type attribute.